src/api/routes/interviews.ts

```diff
@@ -208,13 +209,29 @@ const getInterviewerStats = procedure
       return acc;
     }, {});
 
-  const stats = mentors
-    .filter(mentor => interviewCountMap[mentor.id])
-    .map(mentor => ({
-      user: mentor,
-      interviews: interviewCountMap[mentor.id]
-    }))
-    .sort((i1, i2) => i1.interviews - i2.interviews);
+  const stats = interviewers
+    .filter(interviewer => interviewCountMap[interviewer.id])
+    .map(interviewer => ({
+      user: interviewer,
+      interviews: interviewCountMap[interviewer.id]
+    }));
+  
+  const mentors = await db.User.findAll({
```

Comment:
Why do you need another query? You can already find all the mentors using the role field in the `users` list retrieved in 193. Simply modify the filter at line 213 to include mentors and mentorcoaches as well.